### PR TITLE
[Owners Review]Added missing enums to NI-FGEN

### PR DESCRIPTION
**Pull request by Jha002**
### What does this Pull Request accomplish?

There were lot of enums missing from nimi-python metadata. Hence we have regenerated entire enums.py from base metadata using python script. Additional enums you see in the PR diff are the enums that were missing. 

### Why should this Pull Request be merged?

Missing enums getting generated

### What testing has been done?

Build Passing

### CODE REVIEW

File: source/codegen/metadata/nifgen/enums.py
Comment (Akshata440) [inline source/codegen/metadata/nifgen/enums.py]: We dont need CloseExtCal values since we are not adding support for the API itself

File: source/codegen/metadata/nifgen/enums.py
Comment (Akshata440) [inline source/codegen/metadata/nifgen/enums.py]: We dont need to add this since the attribute using these values `NIFGEN_ATTR_CAL_ADC_INPUT` is not supported

File: source/codegen/metadata/nifgen/CHANGES.md
Comment (Akshata440) [inline source/codegen/metadata/nifgen/CHANGES.md:106]: Please remove all values that are used for performing external calibration

File: source/codegen/metadata/nifgen/attributes.py
Comment (Akshata440) [inline source/codegen/metadata/nifgen/attributes.py]: Why are the attribute changes still seen here? You might have to pull in the latest changes from the feature branch

File: source/codegen/metadata/nifgen/attributes.py
Comment (Akshata440) [inline source/codegen/metadata/nifgen/attributes.py]: This doesnt look like the correct enum name, I think the 'enum' and 'name' tags have been interchanged

File: source/codegen/metadata/nifgen/attributes.py
Comment (Akshata440) [inline source/codegen/metadata/nifgen/attributes.py]: This set of attributes are for calibration. I think merging the latest changes will solve a lot of these discrepancies

**Review by Akshata440 — CHANGES_REQUESTED** [review]

**Review by doshirohan — CHANGES_REQUESTED** [review]
I will wait till Akshata's comments are addressed and merge issues related to attributes.py are resolved.

File: source/codegen/metadata/nifgen/enums.py
Comment (Jha002) [reply source/codegen/metadata/nifgen/enums.py]: removed the enum

File: source/codegen/metadata/nifgen/enums.py
Comment (Jha002) [reply source/codegen/metadata/nifgen/enums.py]: removed the enum

File: source/codegen/metadata/nifgen/attributes.py
Comment (Jha002) [reply source/codegen/metadata/nifgen/attributes.py]: merged with the feature branch

File: source/codegen/metadata/nifgen/attributes.py
Comment (Jha002) [reply source/codegen/metadata/nifgen/attributes.py]: merged with feature branch to remove dicrepancies in attributes.py

File: generated/nifgen/nifgen.proto
Comment (doshirohan) [inline generated/nifgen/nifgen.proto]: Do you need `Parameter` suffix on this enum?

Comment (doshirohan) [reply generated/nifgen/nifgen.proto]: I see that you already have another enum with `FrequencyListHandle` below, but this is not a good way to name the enum. We should find better name for this.

File: generated/nifgen/nifgen_library.cpp
Comment (doshirohan) [inline generated/nifgen/nifgen_library.cpp]: This file and HWS related changes in proto file above should go away after you merge from latest feature branch.

File: source/codegen/metadata/nifgen/enums.py
Comment (doshirohan) [inline source/codegen/metadata/nifgen/enums.py]: Indentation is wrong for few values from here.

File: source/codegen/metadata/nifgen/enums.py
Comment (doshirohan) [inline source/codegen/metadata/nifgen/enums.py]: Fix indentation.

File: source/codegen/metadata/nifgen/enums.py
Comment (doshirohan) [inline source/codegen/metadata/nifgen/enums.py]: Fix indentation.

File: source/codegen/metadata/nifgen/enums.py
Comment (doshirohan) [inline source/codegen/metadata/nifgen/enums.py]: Fix indentation.

File: source/codegen/metadata/nifgen/enums.py
Comment (doshirohan) [inline source/codegen/metadata/nifgen/enums.py]: Fix indentation.

File: source/codegen/metadata/nifgen/enums.py
Comment (doshirohan) [reply source/codegen/metadata/nifgen/enums.py]: I think indentation in general is wrong wherever you have added new values in the enums.py. I have added comments in few places, but fix it everywhere.

File: source/codegen/metadata/nifgen/CHANGES.md
Comment (doshirohan) [reply source/codegen/metadata/nifgen/CHANGES.md:106]: Refresh this list from all the values you have now in `enums.py`. This list has fallen out of sync with what is there in enums.py

**Review by doshirohan — APPROVED** [review]

File: generated/nifgen/nifgen_library.cpp
Comment (Jha002) [reply generated/nifgen/nifgen_library.cpp]: I have merged with the latest feature branch

File: source/codegen/metadata/nifgen/enums.py
Comment (Jha002) [reply source/codegen/metadata/nifgen/enums.py]: fixed it, replaced tabs with whitespaces

**Review by Akshata440 — APPROVED** [review]

File: source/codegen/metadata/nifgen/CHANGES.md
Comment (Jha002) [reply source/codegen/metadata/nifgen/CHANGES.md:106]: updated the list with current version of enums.py

File: generated/nifgen/nifgen.proto
Comment (jonlugoatx) [reply generated/nifgen/nifgen.proto]: I agree `Parameter` isn't the right suffix. Yes, this enum is intended for use as the parameter to one of the functions, but it's a parameter to a function named `ClearFreqListRequest`, not a parameter to a function named `FequencyListHandle`. My recommendation would be to call this `FrequencyListOptions` or `FrequencyListClearOptions`.

**Review by jonlugoatx — APPROVED** [review]
I approve as long as we rename the frequency list enum.

File: source/codegen/metadata/nifgen/CHANGES.md
Comment (doshirohan) [inline source/codegen/metadata/nifgen/CHANGES.md]: Once you rename the enum, make sure to update the name here as well.

File: generated/nifgen/nifgen.proto
Comment (Jha002) [reply generated/nifgen/nifgen.proto]: I have updated the enum with FrequencyListOptions

File: source/codegen/metadata/nifgen/CHANGES.md
Comment (Jha002) [reply source/codegen/metadata/nifgen/CHANGES.md]: I have updated the name in changes.md as well

**Pull request merged**
